applications/GeoMechanicsApplication/custom_elements/axisymmetric_stress_state_policy.cpp

```diff
@@ -0,0 +1,63 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Marjan Fathian
+//
+
+#include "axisymmetric_stress_state_policy.h"
+#include "custom_utilities/element_utilities.hpp"
+
+namespace Kratos
+{
+
+static const SizeType NumberOfUDofPerNode = 2; // Axi-symmetric stress state assumes a 2D problem definition
+
+Matrix AxisymmetricStressState::CalculateBMatrix(const Matrix&         GradNpT,
+                                                       const Vector&         Np,
+                                                       const Geometry<Node>& rGeometry) const
+{
+    const double radius = GeoElementUtilities::CalculateRadius(Np, rGeometry);
```

Comment:
I tried to move the `GeoElementUtilities::CalculateRadius` and `GeoElementUtilities::CalculateAxisymmetricCircumference` to this class, since they are only used for axisymmetric calculations. However, they are also used in (axisymmetric) conditions, and since the `CalculateIntegrationCoefficient` function is slightly different there, I couldn't easily move these calculations to this class